_posts/newsletter/2021-07-08-this-month-in-solid.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: newsletter
+title: This Month in Solid 2021-05-06
+permalink: /newsletter/2021-05-06
```

Comment:
These should be updated to the July date yes?  
Otherwise, it renders with those date in the title:
https://deploy-preview-596--solidproject-dot-org.netlify.app/newsletter/2021-05-06